src/hal/Cargo.toml

```diff
@@ -23,6 +23,8 @@ mint = { version = "0.5", optional = true }
 failure = "0.1"
 serde = { version = "1", features = ["serde_derive"], optional = true }
 fxhash = "0.2.1"
+num-traits = "0.2"
```

Comment:
I don't think it's worth it including the two extra dependencies for `FromPrimitive`, which can be replaced by a simple `transmute`

Reply:
@kvark ok, sounds good! those should be removed now,